cmd/nginx-ingress/flags.go

```diff
@@ -269,6 +275,11 @@ func parseFlags() {
 	if *ingressLink != "" && *externalService != "" {
 		glog.Fatal("ingresslink and external-service cannot both be set")
 	}
+
+	if *enableDynamicSSLReload && !*nginxPlus {
+		glog.V(3).Infof("%s flag requires -nginx-plus and will not be enabled", dynamicSSLReloadParam)
```

Comment:
should this be fatal? similar to maybe:
`*enableAppProtect && !*nginxPlus`

Reply:
Since it is opt-out rather than opt-in we would fail for OSS with fatal, so I think it should stay info?